src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigAligner.java

```diff
@@ -54,56 +48,38 @@ public ContigAligner(final String referenceFilePath) throws IOException {
         log.info("Created BWA MEM");
     }
 
-    public List<AssembledBreakpoint> alignContigs (final ContigsCollection contigsCollection) {
-        final List<AssembledBreakpoint> assembledBreakpoints = new ArrayList<>();
+    /**
+     * Takes a collection of assembled contigs and aligns them to the reference with jBWA. Non-canonical
+     * (secondary) alignments are filtered out, preserving the primary and supplementary alignments.
+     * Within the output list, alignments are sorted first by contig (based on the order in which
+     * the contigs were passed in, and then by their start position on the contig).
+     *
+     * @param assemblyId An identifier for the assembly or set of contigs
+     * @param contigsCollection The set of all canonical (primary or supplementary) alignments for the contigs.
+     * @return
+     */
+    public List<AlignmentRegion> alignContigs(final String assemblyId, final ContigsCollection contigsCollection) {
+        final List<AlignmentRegion> alignedContigs = new ArrayList<>();
```

Comment:
can probably initialize this with capacity contigsCollection.getContents().size()